packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/protocol-level-requirements/index.md

```diff
@@ -35,6 +35,7 @@ For an OIN app, the Authorization Code flow looks like the following:
   * `scope` &mdash; the scopes that are contained in the access token
   * `id_token` &mdash; contains specific info (in JWT format) about the authentication and the end user that is needed by the application
   * Optional. `refresh_token` &mdash; returned if the `offline_access` scope is granted. You can exchange this token for a new access token if the previous access token is expired
```

Comment:
@ScottRussell-okta , @pvershinin-okta - based on OKTA-441197, refresh_token is not supported for OIN OIDC apps with Web mode. Should we remove line 37 since the article is specifically for OIN OIDC with auth flow for web apps? The added Note (below) would confuse users more because the article is for publishing an app on OIN, so we should remove the note as well? Please confirm. Thank you!